bin.js

```diff
@@ -111,7 +111,7 @@ const result = commist()
       process.exit(0)
     }
 
-    authenticate(args.server).then((authToken) => {
+    authenticate(args.server, { useCached: false }).then((authToken) => {
```

Comment:
are we passing useCached: true anywhere?

Reply:
no, it's the default